app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter!
```

Comment:
- - Screenshot reference:

![Selection_055](https://github.com/EvansSnave/Blog_app_rails/assets/107632780/bd1216a1-682e-45d7-8e5f-fcf0d743027b)

- [ ] Just as I pointed out above on the `Users model`, the post counter should have some validation according to the project requirement as i have shown in the screenshot above too. :+1: :+1: I suggest you look at the requirements and add all the validation to the post model. Here is a snippet of the `CommentsCounter must be an integer greater than or equal to zero.` validation could look like. :+1: 

````ruby
validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```